pom.xml

```diff
@@ -92,6 +92,11 @@
         <artifactId>truth</artifactId>
         <version>1.1.5</version>
       </dependency>
+      <dependency>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
+        <version>2.40.0</version>
+      </dependency>
```

Comment:
Is this used? I don't think it has an effect on the plugin configured below.

Reply:
The official [docs](https://github.com/diffplug/spotless/tree/main/plugin-maven#quickstart) say:
> To use it in your pom, just [add the Spotless dependency](https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.diffplug.spotless%22%20AND%20a%3A%22spotless-maven-plugin%22), and configure it like so:

but you’re right, I removed it and everything still works. Good catch!

Reply:
I assume the wording there was just a bit misleading; have created https://github.com/diffplug/spotless/pull/1878 to change that. Let's see what the maintainers of Spotless think.